exonum/src/blockchain/mod.rs

```diff
@@ -439,6 +443,47 @@ impl Blockchain {
         let it = peers_cache.iter().map(|(k, v)| (k, v.clone()));
         it.collect()
     }
+
+    /// Saves the given raw message to the consensus messages cache.
+    pub fn save_message(&mut self, round: Round, raw: &RawMessage) {
```

Comment:
Would it make sense to reduce this method to `save_messages(round, ::std::iter::once(raw))` to reduce code duplication?